### PR TITLE
Relocate test-related files into __tests__ directory

### PR DESCRIPTION
**Pull request by Reallife**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes or No
| New feature?  | Yes or No

### Changed

- Tests-related files was moved into `__tests__` directory

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I wrote unit tests for my code _(if tests is required for my changes)_
- [ ] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,8 +1,6 @@
 * text=auto
 
 /.github          export-ignore
-/sdk/php/tests    export-ignore
-/sdk/php/phpunit* export-ignore
 /*.md             export-ignore
 /Makefile         export-ignore
 /[Dd]ocker*       export-ignore
```
Comment (tarampampam) [inline .gitattributes:6]:
```suggestion
/__tests__       export-ignore
```

File: .gitattributes [hunk 2]
```diff
@@ -1,8 +1,6 @@
 * text=auto
 
 /.github          export-ignore
-/sdk/php/tests    export-ignore
-/sdk/php/phpunit* export-ignore
 /*.md             export-ignore
 /Makefile         export-ignore
```
Comment (tarampampam) [inline .gitattributes:5]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ The format is based on [Keep a Changelog][keepachangelog] and this project adher
 
 ### Changed
 
-- `Makefile` goals `test` and `install` [#137]
+- `Makefile` and `docker-compose.yml` files moved to `__tests__` directory [#137]
 - Info about supported SDKs added into README.md
```
Comment (tarampampam) [inline CHANGELOG.md:18]:
```suggestion
- Tests now located in `__tests__` directory

```

File: .gitignore [hunk 4]
```diff
@@ -2,16 +2,16 @@
 composer.lock
 
 # yarn lock file
-yarn.lock
-yarn-error.log
+__tests__/yarn.lock
```
Comment (tarampampam) [inline .gitignore:5]: может вместо этого создать отдельный `.gitignore` в `./__tests__`?

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (Reallife) [reply .gitignore]: Done

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
LGTM :+1:

**Pull request merged**